languages/elixir/exercises/concept/lists/.docs/introduction.md

```diff
@@ -0,0 +1,36 @@
+Lists are built-in to the Elixir language. They are considered a basic type, denoted by square brackets. Lists may be empty or hold any number of items of any type. For example:
+
+```elixir
+empty_list = []
+one_item_list = [1]
+two_item_list = [1, 2]
+multiple_type_list = [1, :pi, 3.14]
+```
+
+Elixir implements lists as a linked list, where each node stores the reference to the next list. The first item in the list is referred to as the _head_ and the remaining list of items is called the _tail_. We can use this notation in code:
+
+```elixir
+# [1] represented in [head | tail] notation
+[1 | []]
+
+# [1, 2, 3 ]represented in [head | tail] notation
+[1 | [2 | 3 | []]]
+```
+
+We can use _`[head | tail]`_ notation to prepend elements to a list:
+
+```elixir
+# Suppose
+list = [2, 1]
+
+[3, 2, 1] == [3 | list]
+# => true
+```
+
+There are several Elixir Kernel functions for working with lists, including `hd/1`, `tl/1`, and `length/1`. The `in/2` macro indicates whether an item is a member of a list, e.g.
+
+```elixir
```

Comment:
The `config.json` states that this exercise unlocks the `string-literals` concept. Is the below code meant as an introduction of what a string literal is?

Reply:
Yes, I want to introduce strings at a very high level without delving into string functions, because Elixir uses quite a bit of pattern matching concepts with those functions.  

Maybe this needs to briefly introduce strings as a discrete type without saying much more?

Reply:
I'd say the description you use in the `after.md` would be just perfect here:

> [string][string] literals are a sequence of characters surrounded by double quotes